question-service/questionSeeder.js

```diff
@@ -0,0 +1,112 @@
+import Question from "./model/question-model.js"
```

Comment:
Is the question seeder meant for testing? Cause if we are going to use all the questions for the local database won't there be no need to create the production one as this would suffice for both cases?

Reply:
What I meant to say is if it is only for CI and testing, we could just add dummy questions into it.

Reply:
Nope, it's for docker-compose also. So next time you can use docker-compose to run the entire peerprep locally, and it will start a new mongo container. But of course, docker-compose has an option to save the DB data to disk, so it is only needed whenever you want to build a new container, which may happen several times if there are changes to be made.